docs/data/data-grid/column-ordering/column-ordering.md

```diff
@@ -10,13 +10,13 @@ By default, columns are ordered according to the order they are included in the
 
 By default, `DataGridPro` allows all column reordering by dragging the header cells and moving them left or right.
 
-{{"demo": "ColumnOrderingGrid.js", "disableAd": true, "bg": "inline"}}
+{{"demo": "ColumnOrderingGrid.js", "bg": "inline"}}
```

Comment:
The ads on the demos are automatically hidden when the page has the `disableAd` prop. I'm removing the duplication.